File: Arquetipo-Bancolombia-Heroe/domain/model/src/main/java/co/com/sofka/model/tablero/Tablero.java
package co.com.sofka.model.tablero;

import co.com.sofka.model.carta.Carta;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Map;

@Data
@Builder(toBuilder = true)
@NoArgsConstructor
@AllArgsConstructor
public class Tablero {

    private String id;
    private String idjugador;
    private String idcarta;
    private Integer tiempo;
    private String ronda;
    private String idJuego;


    @Override
    public String toString() {
        return "Tablero{" +
                "id='" + id + '\'' +
                ", carta=" + idcarta +
                ", jugador=" + idjugador +
                ", tiempo=" + tiempo +
                ", ronda=" + ronda +
                '}';
    }

}

File: frontend-hero/src/app/components/models/tablero.ts
export class Tablero {
    id?: string;
    idjugador: string;
    idcarta: string;
    tiempo?: number;
    uri:string;
    ronda: string;
    idJuego: string;

    constructor(idjugador:string, idcarta: string, tiempo: number, uri: string,  ronda: string, idJuego: string){
        this.idjugador = idjugador;
        this.idcarta = idcarta;
        this.tiempo = tiempo;
        this.uri = uri;
        this.ronda = ronda;
        this.idJuego = idJuego;

    }
}